packages/design-system/src/interfaces/badge.d.ts

```diff
@@ -0,0 +1,5 @@
+interface IBadgeProps
+  type?: 'primary' | 'light';
+}
+
+export const Badge: React.FC<IBadgeProps>;
```

Comment:
not sure about this (I encountered a lot of problems when tried to use `const` and `React.FC` instead of `ComponentType`, could you check if it's working with the Agent App. You can simply use `npm link`

Reply:
Also, take a look at https://github.com/livechat/design-system/pull/85/files#diff-6d2c4a9bb6e4b0a30e4717337472a95bR2 - previous declaration of types for Badge
Please add `extends React.HTMLAttributes<HTMLSpanElement>` to the IBadgeProps to support span element attributes

Reply:
@kamilmateusiak Why do we need to support span element attributes?

Reply:
Quick example: your tester needs to write some automatic tests (if the badge is appearing) and you need to add data-test attribute to the component). How will you accomplish the task? 
It's a common pattern in the lib. We don't want to limit components to only a few props which we guess that will be needed.

Reply:
`className` is a must here. It would be great to use this badge in `Avatar` component of Agent App. We need to position it correctly (`position: absolute` with a few styles).